tests/acceptance/github-release-test.js

```diff
@@ -0,0 +1,99 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import startApp from 'dummy/tests/helpers/start-app';
+import Pretender from 'pretender';
+import Ember from 'ember';
+
+const { run } = Ember;
+
+let server, app, container, store;
+
+moduleForAcceptance('Acceptance | github release', {
+  beforeEach() {
+    server = new Pretender();
+    server.prepareBody = function (body) {
+      return JSON.stringify(body);
+    };
+    app = startApp();
+    container = app.__container__;
+    store = run(container, 'lookup', 'service:store');
+  },
+
+  afterEach() {
+    server.shutdown();
+    run(app, app.destroy);
+    Ember.BOOTED = false;
+  }
+});
+
+test('finding a release without authorization', function (assert) {
```

Comment:
One more minor thing, actually...move forward, I'd like to have all our test assertions have a short description of the assertion being run, so that individual failures are easier to read from the test results. This is always the last parameter of any `assert` method. If you could add that to your tests, that would be great! I am planning to fix this across the project for any existing tests as well.